pumps/mongo.go

```diff
@@ -58,6 +59,16 @@ type BaseMongoConf struct {
 	MongoDBType                   MongoType `json:"mongo_db_type" mapstructure:"mongo_db_type"`
 }
 
+func (b *BaseMongoConf) GetBlurredURL() string {
```

Comment:
@matiasinsaurralde can we add all the example in the url https://regex101.com/r/E34wQO/1 as unit tests?